packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -182,7 +182,7 @@ class _OffsetScrollbarState extends State<OffsetScrollbar> {
   }
 }
 
-void scrollToPosition(
+void maybeScrollToPosition(
```

Comment:
add dart doc above this method signature. Dart doc is with three slashes `///` and should describe what the method does, as opposed to comments that can be used in the body of a method. e.g.

```
/// Scrolls to [position] if [position] is not already visible in the scroll view.
```